index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile-Version-2.0</title>
+
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="stylesheet" href="styles/header.css">
+  <link rel="stylesheet" href="styles/work-info.css">
+  <link rel="stylesheet" href="styles/gradient.css">
+
+</head>
+<body>
+  <main>
+    <header>
+      <div class="work-container">
+        <div class="vertical-container">
+          <img class="vertical" src="img/vertical.svg" >
+          <div class="arrow-container">
+            <img src="img/arrow.svg" >
+          </div>
+        </div>
+        
+        <p class="work">Work</p>
+      </div>
+      <div class="recent-container">
+        <p class="text-gradient recent">MY RECENT<br> WORK</p>
+      </div>
+
+      <hr class="line">
+    </header>
+
+    <section class="grid-layout">
+      <div class="first-card">
+        <div class="img-place-container left-pos">
+          <img class="img-place" src="img/Img Placeholder.svg" >
+          <div class="ellipse">
+            <img src="img/Ellipse 19.svg">
+          </div>
+        </div>
+        <div class="heading">
+          <p class="work-header">Multi-Post Stories</p>
+        </div>
+        <div class="description">
+          <p class="work-description">A daily selection of privately personalized 
+            reads; no accounts or sign-ups required. has 
+            been the industry's standard dummy text ever 
+            since the 1500s, when an unknown printer took a 
+            standard dummy text.</p>
+        </div>
+
+        <div class="work-skills-container">
+          <ul class="work-skills">
+            <li>CSS</li>
+            <li><img src="img/Rectangle 70.svg"></li>
+            <li>HTML</li>
+            <li><img src="img/Rectangle 70.svg"></li>
+            <li>bootstrap</li>
+            <li><img src="img/Rectangle 70.svg"></li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+
+        <div class="button-container">
+          <button type="button" class="button">See Project</button>
+        </div>
+      </div>
+      <div>
```

Comment:
- [ ] I can see you used a grid on the work section which is good but still there is something you missed out kindly consider checking on this requirement again  👇 
> - In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card. 
 and try to display your single card using grid